easybuild/easyconfigs/g/GLib/GLib-2.34.3-goolf-1.5.14.eb

```diff
@@ -15,6 +15,7 @@ sources = ['glib-%(version)s.tar.xz']
 dependencies = [
     ('libffi', '3.0.13'),
     ('gettext', '0.18.2'),
+    ('libxml2', '2.9.1', '', ('GCC', '4.8.2')),
```

Comment:
hmm, this introduces conflicts higher up the chain:

```
Conflict found for dependencies of PyQt-4.11.3-goolf-1.5.14-Python-2.7.9: zlib-1.2.8-GCC-4.8.2 vs zlib-1.2.8-goolf-1.5.14
```

So, I'll need to go with a full toolchain build of libxml2 here...